src/ref-helper.ts

```diff
@@ -249,7 +249,7 @@ export async function checkCommitInfo(
         `Expected head sha ${expectedHeadSha}; actual head sha ${actualHeadSha}`
       )
       const octokit = github.getOctokit(token, {
-        baseUrl: getServerApiUrl(baseUrl),
+        baseUrl: getServerApiUrl(),
```

Comment:
(same as above)  Biggest open question here: is there a **valid** use case where `baseUrl` might represent some other domain?